core/themes/seven/css/style.css

```diff
@@ -1586,6 +1586,25 @@ body.maintenance-page #logo {
   margin: 0 auto;
 }
 
+/* Links for the simplified login pages. */
+.user-form-links ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.user-form-links li {
+  display: inline;
+  margin: 0.25em 0;
+}
+.user-form-links li:not(:last-child):after {
+  content: "•";
+  margin: 0 1em;
+  color: #666;
```

Comment:
I know, Seven has no Color support, and already "50 shades of grey", but why not using "currentColor" here?